SmartDeviceLink/SDLSecondaryTransportManager.m

```diff
@@ -536,10 +536,15 @@ - (BOOL)sdl_isTCPReady {
     return YES;
 }
 
-#pragma mark - SDLProtocolListener Implementation
+#pragma mark - SDLProtocolDelegate Implementation
 
 // called on transport's thread, notifying that the transport is established
 - (void)onProtocolOpened {
```

Comment:
Would it be better to pass the protocol in all of these messages to do an identity check?